src/com/company/Main.java

```diff
@@ -1,5 +1,6 @@
 package com.company;
 
+import java.security.cert.X509Certificate;
```

Comment:
??? A to co to?

Reply:
ale ja tego nie widzę, testowałam w czasie pracy działanie różnych podpowiedzi, musiało się dodać, ale jak mam to teraz znaleźć?